pkg/cli/tail.go

```diff
@@ -154,10 +161,27 @@ func (c *TailCommand) Flags() *cli.FlagSet {
 			`additional lumberjack specific checks on log labels.`,
 	})
 
+	f.BoolVar(&cli.BoolVar{
+		Name:    "follow",
+		Aliases: []string{"f"},
+		Target:  &c.flagFollow,
+		Default: false,
+		Usage:   "Whether to stream the logs as they show up.",
+	})
+
+	f.IntVar(&cli.IntVar{
+		Name:    "follow-duration",
+		Aliases: []string{"fd"},
+		Default: 3600,
+		Target:  &c.flagFollowDuration,
+		Usage: `the max time in seconds for running -f if there is no new log ` +
```

Comment:
I am not sure if it is the right usage, if it is used in retry.WithMaxRetries, it looks like the max number of retry attempts